app/assets/stylesheets/application.scss

```diff
@@ -173,3 +173,33 @@ table {
     line-height: 1.25;
   }
 }
+
+.banner-dangerous {
+  @include bold-19;
+  background: $white;
+  color: $error-colour;
+  border: 5px solid $error-colour;
```

Comment:
The code isnt wrong, its more the fact that you can use SASS extend and inherit methods to inherit the error styling rather that re-writing the wheel
http://sass-lang.com/guide#topic-7